src/v2/Apps/Home/Components/HomeHeroUnits/HomeHeroUnits.tsx

```diff
@@ -29,6 +30,8 @@ const HomeHeroUnits: React.FC<HomeHeroUnitsProps> = ({ homePage }) => {
         // If logged out, the first hero unit is static and
         // needs to be rendered without the fragment container
         if (!isLoggedIn && i === 0) {
+          if (skipHomeHero) return
```

Comment:
Could you add a comment like `TODO: Remove after <whateverdate>`

Reply:
Done!

Reply:
🙏